### PR TITLE
BHBC-785

### PR DESCRIPTION
**Pull request by cgarrettjones**
<details open>
  <summary><a href="https://quartech.atlassian.net/browse/BHBC-785" title="BHBC-785" target="_blank">BHBC-785</a></summary>
  <br />
  <table>
    <tr>
      <th>Summary</th>
      <td>First pass model the solution for dealing with attachments added to BioHub projects in the first release </td>
    </tr>
    <tr>
      <th>Type</th>
      <td>
        Story
        Story
      </td>
    </tr>
    <tr>
      <th>Status</th>
      <td>In Progress</td>
    </tr>
    <tr>
      <th>Points</th>
      <td>N/A</td>
    </tr>
    <tr>
      <th>Labels</th>
      <td>-</td>
    </tr>
  </table>
</details>

---

# Overview

## Links to jira tickets

- BHBC-785

## This PR contains the following changes

- addition of project_attachment table
- addition of system_constant table
- some name fixing for table and ids around IUCN

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [ ] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [x] Misc cleanup / Refactoring / Documentation
- [x] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [x] I have performed a self-review of my own code

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [x] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Screenshots

Please add any relevant UI screenshots, if applicable.

### CODE REVIEW

File: database/src/migrations/release.0.8/populate_project_type.sql [hunk 1]
```diff
@@ -2,4 +2,4 @@
 insert into project_type (name, record_effective_date) values ('Fisheries', now());
 insert into project_type (name, record_effective_date) values ('Wildlife', now());
 insert into project_type (name, record_effective_date) values ('Aquatic Habitat', now());
-insert into project_type (name, record_effective_date) values ('Terrestrial Habitat', now());
+insert into project_type (name, record_effective_date) values ('Terrestrial Habitat', now());
```
Comment (sdevalapurkar) [inline database/src/migrations/release.0.8/populate_project_type.sql]:
```suggestion
insert into project_type (name, record_effective_date) values ('Terrestrial Habitat', now());

```

File: database/src/migrations/release.0.8/populate_user_identity_source.sql [hunk 2]
```diff
@@ -9,5 +9,4 @@ insert into user_identity_source(name, record_effective_date, description, creat
 insert into system_user (uis_id, user_identifier, record_effective_date, create_date, create_user)
   select id, 'postgres', now(), now(), 1 from user_identity_source 
     where name = 'DATABASE'
-    and record_end_date is null;
-    
+    and record_end_date is null;
```
Comment (sdevalapurkar) [inline database/src/migrations/release.0.8/populate_user_identity_source.sql]:
```suggestion
    and record_end_date is null;
    
```

**Review by sdevalapurkar — DISMISSED** [review]
LGTM, haven't run it locally - but I think it looks good (specifically based on the iucn table updates)

**Review by anissa-agahchen — DISMISSED** [review]
Looks good.

**Review by anissa-agahchen — APPROVED** [review]

**Pull request merged**